exareme2/controller/services/flower/flower_algorithm_registry.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+from enum import Enum
+from enum import unique
+from logging import Logger
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+@unique
+class Status(str, Enum):
+    SUCCESS = "success"
+    RUNNING = "running"
+    FAILURE = "failure"
+
+
+class AlgorithmInputDataDTO(BaseModel):
```

Comment:
Does this need to be here? Doesn't it already exist in `exareme2.controller.services.api.algorithm_request_dtos`?

Reply:
Yeah I temporally removed the dependency for testing nice catch.